diff -ur xmovie-1.6/Makefile xmovie-1.6.new/Makefile
--- xmovie-1.6/Makefile	Wed Feb  7 22:38:46 2001
+++ xmovie-1.6.new/Makefile	Mon Apr 16 04:14:38 2001
@@ -1,16 +1,10 @@
 all:
-	$(MAKE) -C quicktime
-	$(MAKE) -C libmpeg3
-	$(MAKE) -C libsndfile
 	$(MAKE) -C avifile
 	$(MAKE) -C guicast
 	$(MAKE) -C xmovie
 
 clean:
-	$(MAKE) -C quicktime clean
-	$(MAKE) -C libmpeg3 clean
-	$(MAKE) -C libsndfile clean
 	$(MAKE) -C avifile clean
 	$(MAKE) -C guicast clean
 	$(MAKE) -C xmovie clean
 
diff -ur xmovie-1.6/configure xmovie-1.6.new/configure
--- xmovie-1.6/configure	Wed Feb  7 22:38:46 2001
+++ xmovie-1.6.new/configure	Mon Apr 16 04:19:03 2001
@@ -15,7 +15,7 @@
 EOF
 
 if test -z "$CFLAGS"; then
-	echo >> $GLOBAL_CONFIG "CFLAGS = -O2 -funroll-loops -fomit-frame-pointer -malign-loops=2 -malign-jumps=2 -malign-functions=2 -march=i486 -I/usr/local/include"
+	echo >> $GLOBAL_CONFIG "CFLAGS = \$(COPTS)"
 fi
 
 echo >> global_config "CFLAGS += -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64"
@@ -25,19 +25,9 @@
 TARBALL = xmovie-1.4.tar.gz
-CC = g++
+CC = gcc
 GCC = gcc
-CFLAGS += -I../guicast -I../quicktime -I../libmpeg3 -I../libsndfile -I../avifile/include
+CFLAGS += -I../guicast -I/usr/X11R6/include -I/usr/X11R6/include/quicktime -I/usr/X11R6/include/libmpeg3 -I../avifile/include
 LFLAGS = -L/usr/X11R6/lib
-LIBS = ../libmpeg3/libmpeg3.a ../quicktime/libquicktime.a ../guicast/libguicast.a \\
+LIBS = ../guicast/libguicast.a -lmpeg3 -lquicktime -lstdc++ \\
  	-lX11 -lXext -lXv -lpthread -lglib -lm -lpng -lz -ldl
 LDLINKER = g++ -shared 
 EOF
-
-
-cd libmpeg3
-./configure
-cd ..
-
-
-cd quicktime
-./configure
-cd ..
diff -ur xmovie-1.6/guicast/Makefile xmovie-1.6.new/guicast/Makefile
--- xmovie-1.6/guicast/Makefile	Wed Feb  7 22:38:46 2001
+++ xmovie-1.6.new/guicast/Makefile	Mon Apr 16 04:13:34 2001
@@ -35,6 +35,10 @@
 	bcwindow.o \
 	bcwindowbase.o \
 	bcwindowdraw.o \
+	colormodels.o \
+	cmodel_default.o \
+	cmodel_yuv420p.o \
+	cmodel_yuv422.o \
 	errorbox.o \
 	defaults.o \
 	filesystem.o \
@@ -61,7 +62,7 @@
 	$(CC) -o replace replace.o libguicast.a ../quicktime/libquicktime.a $(LFLAGS) $(XLIBS) -lpng
 
 .C.o:
-	$(CC) -c `./c_flags` $*.C
+	$(CC) -c $(CFLAGS) $*.C
 
 clean:
 	rm -f core
diff -ur xmovie-1.6/xmovie/Makefile xmovie-1.6.new/xmovie/Makefile
--- xmovie-1.6/xmovie/Makefile	Wed Feb  7 22:38:46 2001
+++ xmovie-1.6.new/xmovie/Makefile	Mon Apr 16 04:13:34 2001
@@ -46,7 +46,7 @@
 	$(CC) -o test test.o $(LFLAGS) $(LIBS)
 
 .C.o:
-	$(CC) -c `./c_flags` $*.C
+	$(CC) -c $(CFLAGS) $*.C
 
 clean:
 	rm -f core
--- xmovie-1.6/xmovie/Makefile~	Mon Apr 16 04:22:57 2001
+++ xmovie-1.6/xmovie/Makefile	Mon Apr 16 04:25:36 2001
@@ -1,7 +1,6 @@
 AVIDIR = ../avifile
 include ../global_config
 include ../avifile/makefile_objs
-CFLAGS += -g
 
 export CFLAGS
 PREFIX=/usr
@@ -31,14 +28,14 @@
 	xmovie.o
 
 OUTPUT = xmovie
-LIBS = ../libmpeg3/libmpeg3.a \
-	../quicktime/libquicktime.a \
+LIBS = -lmpeg3 \
+	-lquicktime \
 	../guicast/libguicast.a \
-	../libsndfile/libsndfile.a \
+	-lsndfile -lstdc++ \
  	-lX11 -lXext -lXv -lpthread -lglib -lm -lpng -lz -ldl
 
 $(OUTPUT): $(OBJS)
-	$(CC) -o $(OUTPUT) $(OBJS) $(AVIOBJS) $(LFLAGS) $(LIBS)
+	$(CC) -o $(OUTPUT) $(OBJS) $(AVIOBJS) $(LFLAGS) $(LIBS)
 	strip $(OUTPUT)
 
 
