diff -Nur xmovie-1.5.3.orig/Makefile xmovie-1.5.3/Makefile
--- xmovie-1.5.3.orig/Makefile	Wed Nov 29 21:23:07 2000
+++ xmovie-1.5.3/Makefile	Wed Jan  3 02:34:43 2001
@@ -1,12 +1,8 @@
 all:
-	$(MAKE) -C quicktime
-	$(MAKE) -C libmpeg3
 	$(MAKE) -C guicast
 	$(MAKE) -C xmovie
 
 clean:
-	$(MAKE) -C quicktime clean
-	$(MAKE) -C libmpeg3 clean
 	$(MAKE) -C guicast clean
 	$(MAKE) -C xmovie clean;
 
diff -Nur xmovie-1.5.3.orig/configure xmovie-1.5.3/configure
--- xmovie-1.5.3.orig/configure	Wed Nov 29 21:23:07 2000
+++ xmovie-1.5.3/configure	Wed Jan  3 03:02:15 2001
@@ -15,26 +15,16 @@
 EOF
 
 if test -z "$CFLAGS"; then
-	echo >> $GLOBAL_CONFIG "CFLAGS = -O2 -funroll-loops -fomit-frame-pointer -malign-loops=2 -malign-jumps=2 -malign-functions=2 -march=i486 -I/usr/local/include"
+	echo >> $GLOBAL_CONFIG "CFLAGS = \$(COPTS)"
 fi
 
 cat >> $GLOBAL_CONFIG << EOF
 INSTDIR = xmovie
 TARBALL = xmovie-1.4.tar.gz
-CC = g++
-CFLAGS += -I../guicast -I../quicktime -I../libmpeg3
+CC = gcc
+CFLAGS += -I../guicast -I/usr/X11R6/include -I/usr/X11R6/include/quicktime -I/usr/X11R6/include/libmpeg3 -I/usr/lib/glib/include
 LFLAGS = -L/usr/X11R6/lib
-LIBS = ../libmpeg3/libmpeg3.a ../quicktime/libquicktime.a ../guicast/libguicast.a \\
+LIBS = ../guicast/libguicast.a -lmpeg3 -lquicktime \\
  	-lX11 -lXext -lXv -lpthread -lglib -lm -lpng -lz -ldl
 LDLINKER = g++ -shared 
 EOF
-
-
-cd libmpeg3
-./configure
-cd ..
-
-
-cd quicktime
-./configure
-cd ..
diff -Nur xmovie-1.5.3.orig/guicast/Makefile xmovie-1.5.3/guicast/Makefile
--- xmovie-1.5.3.orig/guicast/Makefile	Wed Nov 29 21:23:09 2000
+++ xmovie-1.5.3/guicast/Makefile	Wed Jan  3 02:50:30 2001
@@ -34,6 +34,7 @@
 	bcwindow.o \
 	bcwindowbase.o \
 	bcwindowdraw.o \
+	colormodels.o \
 	errorbox.o \
 	defaults.o \
 	filesystem.o \
@@ -65,7 +66,7 @@
 	$(CC) -o test test.o libguicast.a ../libmpeg3/libmpeg3.a ../quicktime/libquicktime.a $(LFLAGS) $(LIBS) -lpng
 
 .C.o:
-	$(CC) -c `./c_flags` $*.C
+	$(CC) -c $(CFLAGS) $*.C
 
 clean:
 	rm -f core
diff -Nur xmovie-1.5.3.orig/xmovie/Makefile xmovie-1.5.3/xmovie/Makefile
--- xmovie-1.5.3.orig/xmovie/Makefile	Wed Nov 29 21:23:07 2000
+++ xmovie-1.5.3/xmovie/Makefile	Wed Jan  3 02:34:43 2001
@@ -33,7 +33,7 @@
 	$(CC) -o test test.o $(LFLAGS) $(LIBS)
 
 .C.o:
-	$(CC) -c `./c_flags` $*.C
+	$(CC) -c $(CFLAGS) $*.C
 
 clean:
 	rm -f core
