--- xmovie-1.9.7/xmovie/filesndfile.C.orig	Sun Jun  9 14:34:38 2002
+++ xmovie-1.9.7/xmovie/filesndfile.C	Sun Oct  6 13:04:51 2002
@@ -22,7 +22,7 @@
 	
 	bzero(&fd_config, sizeof(SF_INFO));
 //printf("FileSndFile::check_sig 2\n");
-	fd = sf_open_read(asset->path, &fd_config);
+	fd = sf_open(asset->path, SFM_READ, &fd_config);
 //printf("FileSndFile::check_sig 3\n");
 
 	if(fd)
@@ -52,7 +52,7 @@
 //printf("FileSndFile::open_file 1\n");
 	bzero(&fd_config, sizeof(SF_INFO));
 //printf("FileSndFile::open_file 1\n");
-	if(!(fd = sf_open_read(asset->path, &fd_config)))
+	if(!(fd = sf_open(asset->path, SFM_READ, &fd_config)))
 	{
 		printf("FileSndFile::open_file\n");
 		return 1;
@@ -80,12 +80,12 @@
 
 long FileSndFile::get_audio_length()
 {
-	return fd_config.samples;
+	return fd_config.frames;
 }
 
 int FileSndFile::end_of_audio()
 {
-	return file->audio_position >= fd_config.samples;
+	return file->audio_position >= fd_config.frames;
 }
 
 int FileSndFile::read_audio(char *buffer, long len)
