--- xmovie-1.9.8/build/Makefile.xmovie.orig	Mon Feb 10 04:12:18 2003
+++ xmovie-1.9.8/build/Makefile.xmovie	Fri Feb 28 01:25:13 2003
@@ -1,9 +1,6 @@
 include global_config
 
 DIRS := \
-	libmpeg3 \
-	libsndfile \
-	quicktime \
 	guicast \
 	xmovie \
 
--- xmovie-1.9.8/xmovie/Makefile.orig	Wed Jan 29 07:33:17 2003
+++ xmovie-1.9.8/xmovie/Makefile	Fri Feb 28 01:29:45 2003
@@ -31,15 +31,14 @@
 OUTPUT = $(OBJDIR)/xmovie
 
 CFLAGS += -I../guicast \
-	-I../quicktime \
-	-I../libmpeg3 \
-	-I../libsndfile
+	-I/usr/include/quicktime \
+	-I/usr/include/libmpeg3
 
 LIBS += -L/usr/X11R6/lib \
-	../libmpeg3/$(OBJDIR)/libmpeg3.a \
-	../quicktime/$(OBJDIR)/libquicktime.a \
+	-lmpeg3 \
+	-lquicktime \
 	../guicast/$(OBJDIR)/libguicast.a \
-	../libsndfile/$(OBJDIR)/libsndfile.a \
+	-lsndfile \
  	$(GLLIBS) -lX11 -lXext -lXv -lpthread -lglib -lm -lpng -lz -ldl $(XXF86VM)
 
 # Static libraries for distribution
@@ -68,7 +67,6 @@
 	cat *.C *.h | wc
 
 install:
-	strip $(OUTPUT)
 	rm -f $(PREFIX)/bin/xmovie
 	cp $(OUTPUT) $(PREFIX)/bin
 
--- xmovie-1.9.8/guicast/Makefile.orig	Wed Jan 29 07:32:41 2003
+++ xmovie-1.9.8/guicast/Makefile	Fri Feb 28 01:34:45 2003
@@ -1,6 +1,6 @@
 include ../global_config
 
-CFLAGS += -I../quicktime
+CFLAGS += -I/usr/include/quicktime
 
 
 $(shell sh -c 'if ! test -d $(OBJDIR)\; then mkdir $(OBJDIR)\; fi' )
