diff -burN xmovie-1.9.12.orig/build/Makefile.xmovie xmovie-1.9.12/build/Makefile.xmovie
--- xmovie-1.9.12.orig/build/Makefile.xmovie	2004-03-06 14:21:43.670738960 +0100
+++ xmovie-1.9.12/build/Makefile.xmovie	2004-03-06 14:23:00.384076760 +0100
@@ -2,21 +2,11 @@
 
 # system tests
 
-SNDFILE_DIR := $(shell expr libsndfile* )
-
-
-
 DIRS := \
-	libmpeg3 \
-	quicktime \
 	guicast \
 	xmovie \
 
-
-
-
 all:
-	$(MAKE) -f build/Makefile.sndfile
 	@ for i in $(DIRS) ; \
 	do \
 		 $(MAKE) -C $$i ; \
diff -burN xmovie-1.9.12.orig/guicast/Makefile xmovie-1.9.12/guicast/Makefile
--- xmovie-1.9.12.orig/guicast/Makefile	2004-03-06 14:21:51.723514752 +0100
+++ xmovie-1.9.12/guicast/Makefile	2004-03-06 14:23:35.013812240 +0100
@@ -2,9 +2,8 @@
 
 # Need the freetype include that matches the X server for Xft rendering
 #FREETYPE_DIR := ..//usr/X11R6/include/freetype2
-FREETYPE_DIR := $(shell expr ../freetype*/include )
 
-CFLAGS += -I../quicktime -I$(FREETYPE_DIR)
+CFLAGS += -I/usr/include/quicktime -I/usr/include/freetype2
 
 # must only define here so applications using libpng don't see any of the XFT
 # includes at all.  They use a conflicting setjmp.
diff -burN xmovie-1.9.12.orig/xmovie/Makefile xmovie-1.9.12/xmovie/Makefile
--- xmovie-1.9.12.orig/xmovie/Makefile	2004-03-06 14:21:56.603772840 +0100
+++ xmovie-1.9.12/xmovie/Makefile	2004-03-06 14:59:56.871119512 +0100
@@ -1,7 +1,6 @@
 #AVIDIR = ../avifile
 #include ../avifile/makefile_objs
 include ../global_config
-SNDFILE_DIR := $(shell expr ../libsndfile* )
 
 PREFIX=/usr
 
@@ -32,15 +31,14 @@
 OUTPUT = $(OBJDIR)/xmovie
 
 CFLAGS += -I../guicast \
-	-I../quicktime \
-	-I../libmpeg3 \
-	-I$(SNDFILE_DIR)/src
+	  -I/usr/include/quicktime \
+	  -I/usr/include/libmpeg3
 
 LIBS += -L/usr/X11R6/lib \
-	../libmpeg3/$(OBJDIR)/libmpeg3.a \
-	../quicktime/$(OBJDIR)/libquicktime.a \
+	-lmpeg3 \
+	-lquicktime \
 	../guicast/$(OBJDIR)/libguicast.a \
-	$(SNDFILE_DIR)/$(OBJDIR)/libsndfile.a \
+	-lsndfile \
  	$(GLLIBS) \
 	-lX11 \
 	-lXext \
@@ -79,7 +77,6 @@
 	cat *.C *.h | wc
 
 install:
-	strip $(OUTPUT)
 	rm -f $(PREFIX)/bin/xmovie
 	cp $(OUTPUT) $(PREFIX)/bin
 
